### ISSUE TITLE
The forward function of a model

### ISSUE THREAD

**Issue by huanghoujing** [Cysu/open-reid#57]
Hi, Tong Xiao.

In the `forward` function of `ResNet` or `InceptionNet`, I am confused by the following two lines:

```python
if self.norm:
    x = F.normalize(x)
elif self.has_embedding:
    x = F.relu(x)
```

Why should `normalize` and `relu` be exclusive to each other? Should the `elif` here be `if` instead?

Waiting for your response. Thanks a lot!

**Comment by Cysu** [comment]
Normalization will make features unit vector. ReLU will violate such constraints.

**Comment by huanghoujing** [comment]
Thank you! I understand it now.
